liblangutil/SourceLocation.h

```diff
@@ -109,13 +117,13 @@ SourceLocation const parseSourceLocation(std::string const& _input, std::string
 /// Stream output for Location (used e.g. in boost exceptions).
 inline std::ostream& operator<<(std::ostream& _out, SourceLocation const& _location)
 {
-	if (_location.isEmpty())
+	if (!_location.isValid())
 		return _out << "NO_LOCATION_SPECIFIED";
 
 	if (_location.source)
 		_out << _location.source->name();
 
-	_out << "[" << _location.start << "," << _location.end << ")";
```

Comment:
The idea here is that "end" is excluded, but maybe it is better like this.

Reply:
Oh, I think `[ ... )` was OK. Reverted.